CDP4CrossViewEditor/ViewModels/CrossViewDialogViewModel.cs

```diff
@@ -100,6 +103,11 @@ private WorkbookRowViewModel SelectedWorkbook
             set => this.RaiseAndSetIfChanged(ref this.selectedWorkbook, value);
         }
 
+        /// <summary>
+        ///
+        /// </summary>
+        private Dictionary<string, string> ManuallyFilledValues = new Dictionary<string, string>();
```

Comment:
See other comments on MR about the name consistency for this dictionary present in 3/4 separate locations.